nx.json

```diff
@@ -12,9 +12,15 @@
   },
   "tasksRunnerOptions": {
     "default": {
-      "runner": "@nrwl/workspace/tasks-runners/default",
+      "runner": "@nrwl/nx-cloud",
       "options": {
-        "cacheableOperations": ["build", "lint", "test", "e2e"]
+        "cacheableOperations": [
+          "build",
+          "lint",
+          "test",
+          "e2e"
+        ],
+        "accessToken": "Y2NhMWI0ZDYtM2IxYS00MzZlLThiNzctNTYwOGRhMjM2YTgwfHJlYWQtd3JpdGU="
```

Comment:
Where is this accessToken from

Reply:
@srleecode looks like it is tied to a workspace within my https://nx.app account...
![image](https://user-images.githubusercontent.com/707567/143668064-33ccbd3f-0446-428a-82f2-63cc46fe35ae.png)

I believe Nx Cloud is free for Open Source projects. I'll check docs to see if I could have done this better.